pkg/container-collection/options.go

```diff
@@ -432,3 +438,280 @@ func WithLinuxNamespaceEnrichment() ContainerCollectionOption {
 		return nil
 	}
 }
+
+// WithKubernetesOwnerRefEnrichment automatically adds owner references metadata
+func WithKubernetesOwnerRefEnrichment(nodeName string) ContainerCollectionOption {
```

Comment:
I don't see why it should be a different enricher. I think it can be integrated in `WithKubernetesEnrichment`.

Reply:
I agree. I added this enrichment into the `WithKubernetesEnrichment` ensuring it is called in all the cases.